src/broker/cas_cgw.c

```diff
@@ -163,22 +172,14 @@ cgw_database_connect (SUPPORTED_DBMS_TYPE dbms_type, const char *connect_url)
 						out_connect_str,
 						(SQLSMALLINT) sizeof (out_connect_str),
 						&out_connect_str_len, SQL_DRIVER_NOPROMPT));
-
+      is_connet = true;
```

Comment:
What about use more meaningful name:
**is_connected**

Reply:
Thanks for your comments. Changed to is_connected.